File: easy/min_deletion_size.go
package easy

func MinDeletionSize(strs []string) int {
	count := 0
	rows := len(strs)
	cols := len(strs[0])
	for col := 0; col < cols; col++ {
		for row := 1; row < rows; row++ {
			if strs[row-1][col] > strs[row][col] {
				count++
				break
			}
		}
	}
	return count
}

File: src/easy/MinDeletionSize.swift
func minDeletionSize(_ strs: [String]) -> Int {
  var count = 0
  let rows = strs.count
  let cols = strs[0].count
  let grid = strs.map { Array($0) }
  for col in 0..<cols {
    for row in 1..<rows {
      if grid[row - 1][col] > grid[row][col] {
        count += 1
        break
      }
    }
  }
  return count
}
